src/styles/brackets_fonts.less

```diff
@@ -80,4 +80,5 @@
 
 /* Font Stacks */
 
-@sansFontFamily: 'SourceSansPro', Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
+@sansFontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
+@sourceFontFamily: "Menlo Regular", Consolas, Inconsolata, "Vera Sans", "Lucida Console", Courier, fixed;
```

Comment:
NOTE: I've got a new commit on https://github.com/adobe/brackets/pull/10727 that will do the `-Medium` bit, but it hasn't landed yet.